### ISSUE TITLE
A11y_Azure Tools Storage Explorer_AddaresourceviaAzureActiveDirectory_ConnectionSummary_Screenreader : The "Warning" icon is not provided with proper alternative text for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/AzureStorageExplorer#3503]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#Azure Tools Storage Explorer;#WCAG1.3.1;#WCAG1.1.1;#MAC;#A11ySev2;#Benchmark;#Screenreader;#VoiceOver;#FTP;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.15.0
AzCopy Version: 10.6.0
OS Version: macOS Catalina Version 10.15.6
Screen Reader: Voiceover

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Add a resource via Azure Active Directory(Azure AD)" radio button and hit enter on Next button.
4. "Attach with azure AD" screen opened and navigate to Next button and hit enter.
5. Enter the required details in edit boxes and hit enter on "Next" button.
6. The "Connection Summary" screen will appear on the screen the start screen reader.
7. Navigate to the "Warning" icon and observe if it is announced with proper alternative text or not.

### Actual:
When screen reader users navigate using 'Caps + arrow keys' through the "Connection Summary" screen, the "Warning" icon is not announced with proper alternative text for screen reader users which is incorrect.

### Expected:
The "Warning" icon should be announced with proper alternative text for screen reader users when they navigate to this icon.

### User Impact:
If proper information is not conveyed to the screen reader users regarding the content present on the screen then the users will not know how to interact and this will lead to confusion for the screen reader users.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-manai_microsoft_com/EbtTT8MmXF9Pn7KPDbZgJjEBUoJECD_A8xm8aT9tnkCHAw?e=gy25iu

**Comment by craxal** [comment]
Conflicts somewhat with #3510. Since this icon isn't purely a decorative image, we should provide alternative text for this image.

**Comment by Ashutoshsahoo164** [comment]
GitHubTags : #Closed
